src/SDK/Trace/Event.php

```diff
@@ -14,11 +13,11 @@ final class Event implements EventInterface
     private int $timestamp;
     private AttributesInterface $attributes;
 
-    public function __construct(string $name, int $timestamp, AttributesInterface $attributes = null)
+    public function __construct(string $name, int $timestamp, AttributesInterface $attributes)
```

Comment:
The [spec](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/api.md#add-events) says "zero or more attributes" for an event, so I think they should remain optional?

Reply:
Callers can provide an empty `AttributesInterface` instance; allowing `null` to represent empty attributes felt unecessary as the only caller (`Span`) always passes non-null attributes.